packages/core/src/builder.class.ts

```diff
@@ -489,6 +491,14 @@ export class Builder {
     this.actions.push(action);
   }
 
+  static registerTrustedHost(host: string) {
+    this.trustedHosts.push(host);
+  }
+
+  static isTrusted(host: string) {
```

Comment:
```suggestion
  static isTrustedHost(host: string) {
```

nit pick, just a little more consistent and clear imo